app/ios/Podfile.lock

```diff
@@ -3,27 +3,27 @@ PODS:
   - path_provider_foundation (0.0.1):
     - Flutter
     - FlutterMacOS
-  - sqflite (0.0.3):
+  - sqflite_darwin (0.0.4):
```

Comment:
Hmm I'm curious which package we have that uses `sqflite` under the hood

Reply:
the sqflite package is being used under the hood by flutter_cache_manager, which is indirectly brought in via cached_network_image.

```
cached_network_image 3.4.1
└── flutter_cache_manager 3.4.1
    └── sqflite 2.4.1
```